src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/integration/health/HealthCheckTest.kt

```diff
@@ -32,14 +59,6 @@ class HealthCheckTest : IntegrationTestBase() {
       )
   }
 
-  @Test
-  fun `Health page reports database ok`() {
```

Comment:
Seems like a good test - wondering why it's removed.

Reply:
I moved it to be part of the overall health OK test. I did wonder about having an OK test for each dependency but went with following the pattern from the template PR

Reply:
Nah that seems fine